ConsoleCalculator/Calculator.cs

```diff
@@ -5,152 +5,152 @@ namespace ConsoleCalculator
                 if (char.IsDigit(key))
                 {
-                    if (operatorPressed == 0 )
+                    if (_operatorPressed == 0 )
                     {
-                        if (!(firstNumber.Equals("0") && key=='0'))
+                        if (!(_firstNumber.Equals("0") && key=='0'))
                         {
-                            firstNumber += key;
+                            _firstNumber += key;
                         }
                         
                                                
                     }
                     else
                     {
-                        if (!(secondNumber.Equals("0")&&key=='0'))
+                        if (!(_secondNumber.Equals("0")&&key=='0'))
                         {
-                            secondNumber += key;
+                            _secondNumber += key;
                         }
                         
-                        return secondNumber;
+                        return _secondNumber;
                     }
                     
                 }
                 else if (key.Equals('.'))
                 {
-                    if (operatorPressed == 0 && decFirst==false)
+                    if (_operatorPressed == 0 && _decimalFirst==false)
                     {
-                        if (firstNumber.Length == 0)
+                        if (_firstNumber.Length == 0)
                         {
-                            firstNumber = "0.";
+                            _firstNumber = "0.";
                         }
                         else
                         {
-                            firstNumber += ".";
+                            _firstNumber += ".";
                         }
-                        decFirst = true;
+                        _decimalFirst = true;
                     }
-                    else if (operatorPressed == 1 && decSecond == false)
+                    else if (_operatorPressed == 1 && _decimalSecond == false)
                     {
-                        if (secondNumber.Length == 0)
+                        if (_secondNumber.Length == 0)
                         {
-                            secondNumber = "0.";
+                            _secondNumber = "0.";
                         }
                         else
                         {
-                            secondNumber += ".";
+                            _secondNumber += ".";
                         }
-                        decSecond = true;
+                        _decimalSecond = true;
                     }
                 }
                 else if (key == 's' || key == 'S')
                 {
-                    if (operatorPressed == 1)
+                    if (_operatorPressed == 1)
                     {
-                        secondNumber = (-double.Parse(secondNumber)).ToString();
+                        _secondNumber = (-double.Parse(_secondNumber)).ToString();

```

Comment:
try to add comments where the 3rd person may find it difficult to understand the code. You should add comments on if conditions or any logic where the code itself is not self-explanatory. I  didn't understand why you added that minus sign in above line of code.